tests/trainer/test_trainer_fsdp.py

```diff
@@ -72,14 +72,15 @@ def test_trainer(self):
         cmd = [
             "accelerate",
             "launch",
-            "--use_fsdp",
             "--main_process_port",
             f"{get_torch_dist_unique_port()}",
             "--num_processes",
             f"{torch.cuda.device_count()}",
+            f"{self.test_file_dir}/test_trainer_fsdp.py",
+            "--fsdp",
+            "full_shard",
             "--fsdp_transformer_layer_cls_to_wrap",
             "GPT2Block",
-            f"{self.test_file_dir}/test_trainer_fsdp.py",
             "--output_dir",
             f"{output_dir}",
```

Comment:
Refactoring the test this way forces us to use the transformers parser rather than accelerate's CLI env var setting